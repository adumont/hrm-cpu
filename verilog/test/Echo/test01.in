5 08
5 06
5 02
10 31
2 32 
3 33
4 34
