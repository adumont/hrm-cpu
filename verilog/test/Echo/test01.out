08
06
02
31
32
33
34
